sdk/azurestackhci/Azure.ResourceManager.Hci/src/Generated/Models/ArcExtensionData.Serialization.cs

```diff
@@ -68,39 +68,6 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             }
             writer.WriteEndObject();
             writer.WriteEndObject();
-            writer.WritePropertyName("systemData");
```

Comment:
before this PR, "systemData" sent to wire in request though it's defined as readonly in swagger.